### PR TITLE
[WIP] Matrix valued

### PR DESCRIPTION
**Pull request by TiKeil**
This is for matrix valued computations in gridlod

### CODE REVIEW

**Comment by fredrikhellman** [comment]
Seems like ecworkerMatrixValued.py and eccontrollerMatrixValued.py are missing. Could you please add them?

File: pg.py [hunk 1]
```diff
@@ -59,7 +62,8 @@ def updateCorrectors(self, coefficient, clearFineQuantities=True, MatrixValued=F
         if self.printLevel >= 2:
             print 'Setting up workers'
         if MatrixValued:
-            eccontrollerMatrixValued.setupWorker(world, coefficient, IPatchGenerator, k, clearFineQuantities, self.printLevel)
+            pass
+            #eccontrollerMatrixValued.setupWorker(world, coefficient, IPatchGenerator, k, clearFineQuantities, self.printLevel)
```
Comment (fredrikhellman) [inline pg.py]: This will need to be fixed when eccontrollerMatrixValued is available

**Comment by TiKeil** [comment]
Oh sorry, I will add it as soon as possible.

File: pg.py (hunk 1 above)
Comment (TiKeil) [reply pg.py]: It's available now

**Pull request merged**